islandora_scholar.drush.inc

```diff
@@ -118,3 +134,147 @@ EOQ;
   $sandbox['offset'] += $citation_update;
   $context['finished'] = $sandbox['offset'] / $sandbox['total'];
 }
+
+
+/**
+ * Command callback to update citations with HTML in MODS.
+ */
+function drush_islandora_scholar_fix_doi_html_in_mods() {
+  $action = drush_get_option('action', variable_get('islandora_doi_handle_imported_metadata', FALSE));
+  if (!in_array($action, array('strip', 'encode'))) {
+    drush_print("This function requires an option, since the GUI variable of how
+     to handle DOI imports is not set. Please use `--action=encode` or
+     `--action=strip`. For more information use `drush help islandora-scholar-fix-doi-html-in-mods`.");
+    exit;
+  }
+  $warnings = dt("WARNING: You are about to irrevocably alter datastream " .
+  "content in place. Backups of modified datastreams, pre-alterations, " .
+  "will be stored in Drupal's temporary:// directory. \n");
+  $warnings .= ($action == 'strip') ?
+  dt("You have chosen to strip errant HTML tags - this will result in data loss.\n") :
```

Comment:
I would probably write the less ominous sounding
```php
  dt("You have chosen to strip errant HTML tags - this will result in the loss of markup information.\n") :
```